compiler/README.md

```diff
@@ -23,33 +28,71 @@ To get started, simply run `./compile --writeKey=<YOUR WRITE KEY>`.
-Segment account associated with the `--writeKey` option, you must enable and set up each integration you'd like to test in your Segment dashboard 
-before building a version of A.js locally using this tool.
- - Once A.js is built, this tool saves a copy of your A.js build in the `./builds` folder. Build names follow the pattern `analytics-${date}.js`, where 
+- This tool builds all A.js integration packages from the `../integrations`
+directory into a local version of A.js.
+- The tool retrieves integration
+settings from `http://cdn.segment.com/v1/projects/${writeKey}/settings`.
+Because we pull settings from the Segment account associated with the
+`--writeKey` option, you must enable and set up each integration you'd like to
+test in your Segment dashboard before building a version of A.js locally using
+this tool.
+- If a custom settings file is specified, the tool uses only those
+integrations and settings defined in the file. An integrations whose
+settings are provided in a file need not be enabled in your production source.
+- Once A.js is built, this tool saves a copy of your A.js build in the
+`./builds` folder. Build names follow the pattern `analytics-${date}.js`, where
 `date` is the unix timestamp in seconds at which the file was generated.
- - Next, the tool launches your default browser, serving a sample `index` file from `./src/index.html`. This file includes the Segment snippet and 
-initializes the your just-generated A.js build so you can begin testing right away.
+- Finally, the tool launches your default browser, serving a sample `index` file
+from `./src/index.html`. This file includes the Segment snippet and initializes
+the your just-generated A.js build so you can begin testing right away.
 
 # Adding New Destinations
 
-Adding new destinations is easy - just add the package name and file path to the `./package.json` file in this directory. For example, to
-add a new integration with slug `google-v2`, just add the following line to the `dependencies` section of `./package.json`:
+Adding new destinations is easy: Just add the package name and file path to
+the `./package.json` file in this directory. For example, to add a new
+integration with slug `google-v2`, add the following line to the
+`dependencies` section of `./package.json`:
 
 ```
 "@segment/analytics.js-integrations-google-v2": "file:../integrations/google-v2"
 ```
 
-Note the `google-v2` destination directory must exist in `../integrations`, and the package name in that directory must match the package name
-specified in the `package.json` file in this directory.
+Note the `google-v2` destination directory must exist in `../integrations`, and
+the package name in that directory must match the package name specified in the
+`package.json` file in this directory.
+
+# Custom Settings
+
+An example settings file is included in `./src/settings_example.json`. Settings
+must be defined as JSON in this format:
+
+```json
+{
+  "integrations": {
```

Comment:
Would be possible to include a _semi_ realistic example? like:
```json
"Google Analytics": {
  "apiToken": "foobar"
},
"Integration Name": {
  "settingSlug": "value"
}
```

Reply:
lol. yes!